api/prisma/schema.prisma

```diff
@@ -57,6 +57,15 @@ model WorksheetCourses {
   @@index([net_id], map: "net_id")
 }
 
+model WorksheetNames {
+  net_id           String @db.Char(8)
+  worksheet_number Int    @db.UnsignedMediumInt
+  name             String @db.Char(255)
+
+  @@unique([net_id, worksheet_number], map: "net_id_worksheet_number")
+  @@index([net_id], map: "net_id")
+}
```

Comment:
I don't know much about Prisma so I can't comment on this, but this looks weird to me: shouldn't the worksheets data structure just be one `name -> courses` map? Why do we do `name -> index -> courses`?

Reply:
After some more thought, I think this makes more sense.